android/library/src/main/java/com/polidea/multiplatformbleadapter/RefreshGattMoment.java

```diff
@@ -10,4 +10,11 @@
     RefreshGattMoment(String name) {
         this.name = name;
     }
+
+    public static RefreshGattMoment getByName(String name) {
```

Comment:
There is already submitted issue to delete this enum because its useless.
Yes, I totally agree, throwing an exception here or informing that it can return null is a good practice, but do we want to do this at this moment? It will be removed in the near future during the first refactor.